src/Cache/CacheEngine.php

```diff
@@ -341,7 +341,7 @@ protected function _key(string $key): string
 
         $prefix = '';
         if ($this->_groupPrefix) {
-            $prefix = md5(implode('_', $this->groups()));
+            $prefix = hash('xxh128', implode('_', $this->groups()));
```

Comment:
This change will rotate cache keys of any cache engine that is using group prefixes. If we change this we'll need to include a note in the release notes.

Reply:
Yes, hence I targeted 5.1.